apps/create-fuels-counter-guide/fuel-toolchain.toml

```diff
@@ -1,2 +1,6 @@
 [toolchain]
 channel = "testnet"
+
+[components]
+forc = "0.66.5"
+fuel-core = "0.40.0"
```

Comment:
Shouldn't this be `0.40.1` tho?

```suggestion
fuel-core = "0.40.1"
```
![testnet](https://github.com/user-attachments/assets/387d8d49-5908-446e-8ecb-c01d6bcc9fa3)

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3475/commits/f18f9989fe6b2b126a998f458e27c746c478e855